lib/typinggame_server/entities/players.rb

```diff
@@ -0,0 +1,2 @@
+class Players < Hanami::Entity
```

Comment:
We probably want this to be `Player` (singular)

Reply:
@jgayfer fixed